.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,51 @@
+name: '🐛 | Bug Report'
+description: >
+  File a bug report
+title: '[🐛 | Bug]: '
+labels: [bug, triage]
+assignees:
+  - touilleMan
```

Comment:
but why meeeee ? :'(

Reply:
because ?

Reply:
So do you want to include another person to be able to do the triage of the bug ?

Reply:
remove the field assignees